example/src/screens/BasicPlayback.tsx

```diff
@@ -22,6 +25,12 @@ export default function BasicPlayback() {
     },
   });
 
+  const config: PlayerViewConfig = {
+    uiConfig: {
+      variant: Platform.isTV ? new TvUi() : new SmallScreenUi(),
+    },
+  };
+
```

Comment:
Yeah either that or extra sample for TVs makes sense to me. I'm leaning more towards just doing it for all the samples. What do you think @strangesource? Would you still prefer an extra sample?

Reply:
I think an extra sample would make the most sense. Customers are unlikely to ever develop an app that runs either on mobile or the TV so it would be more realistic. 
In addition we could make sure that other settings reflect TV use cases as well such as e.g. casting not being enabled.

Reply:
I just had a customer visit where they cover all platforms - even web based platforms - with a single RN code base.

Additionally all of our existing samples are set up for tvOS compatibility already.

Reply:
I will create a follow up, most probably with an extra TV sample. It should be clear enough what's needed to make it work + there are other configurations like disabling casting to make the playback work on the TVs.